rpki-validator/src/main/java/net/ripe/rpki/validator3/api/bgp/BgpRisDownloader.java

```diff
@@ -86,6 +89,9 @@ public void preDestory() throws Exception {
 
     public BgpRisDump fetch(@NotNull BgpRisDump dump) {
         log.info("attempting to download new BGP RIS preview dump from {}", dump.url);
+        long before = System.currentTimeMillis();
```

Comment:
Just in case: there's net.ripe.rpki.validator3.util.Time class, maybe it can make the code simpler here.

Reply:
It's a nice utility (and I had not considered using it here), but I think that this is not convenient here (since http statuses are communicated through exceptions).